sdk/servicebus/azure-servicebus/tests/perf_tests/receive_queue_message_batch.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import asyncio
+
+from ._test_base import _QueueReceiveTest
+
+
+class ReceiveQueueMessageBatchTest(_QueueReceiveTest):
+    def run_sync(self) -> None:
+        count = 0
+        while count < self.args.num_messages:
```

Comment:
I'm wondering if this pattern fits better with the BatchPerfTest, where instead of looping, we could do one `receive_messages` operations and return the number received...